provisioner/terraform/executor.go

```diff
@@ -28,7 +28,10 @@ import (
 	"github.com/coder/coder/v2/provisionersdk/proto"
 )
 
-var version170 = version.Must(version.NewVersion("1.7.0"))
+var (
+	version170 = version.Must(version.NewVersion("1.7.0"))
+	version190 = version.Must(version.NewVersion("1.9.0"))
+)
```

Comment:
Terraform versions below 1.12 are already EOL. We also bundle Terraform in the official image. We may not need to worry about supporting older versions. 
https://endoflife.date/terraform